test/plugins/test_super_check.py

```diff
@@ -33,7 +33,7 @@ def {method}(xyzzy):
     ).format(method=method)
     messages = run_pylint(source, MSG_IDS)
     expected = {
-        "A:super-method-not-called:super(...).{}() not called (unittest.case.TestCase)".format(
+        "14:non-parent-method-called:{}() was called from a non-parent class (unittest.case.TestCase)".format(
```

Comment:
Couldn't find changelog for this fix. Updated according to the error trace in the test failure.